src/notebook_utils/automation.py

```diff
@@ -14,6 +14,67 @@
 def is_ipynb_checkpoint(notebook_file: PathLike):
     return ".ipynb_checkpoint" in str(notebook_file)
 
+def run_notebook(notebook_path: PathLike):
+    return subprocess.run(["jupyter", "nbconvert", "--to", "notebook", "--inplace", "--execute", "--ExecutePreprocessor.timeout=600", f"{notebook_path}"], capture_output=True, text=True)
+
+def run_notebook_cli():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-f",
+        "--file",
+        type=pathlib.Path,
```

Comment:
Is there a `File` type? I vaguely remember seeing that in docs to be more specific about the type of path allowed.

Reply:
Not from what I see online... Only functions to check if the path is a file or a directory (`Path.is_file()` or `Path.is_dir()`.

Reply:
Ok. What I saw was similar to what you did with `ValidateIsDirectory`.